monitors/CRMonitor.js

```diff
@@ -35,7 +35,7 @@ class CRMonitor {
   }
 
   // Queries all monitored wallet ballance for collateralization ratio against a given threshold.
-  checkWalletCrRatio = async () => {
+  async checkWalletCrRatio() {
```

Comment:
Did we decide to not use arrow functions? Maybe add some documentation about this to `CONTRIBUTING.md` in a future PR.

Reply:
See #1545 for this style guide change.